src/pages/home/components/home-ranking/item/HomeRankingItem.tsx

```diff
@@ -0,0 +1,160 @@
+import { RankingWithJWTResponse } from '@/types/ranking';
+import * as S from './HomeRankingItem.style';
+import { useAuthStore } from '@/stores/auth.store';
+
+interface RankingItemProps {
+  data: RankingWithJWTResponse;
```

Comment:
```suggestion
  data: RankingWithJWTResponse | RankingResponse;
```

이 부분도 2가지 타입으로 지정해야 할 것 같아요